tests/fiaas_deploy_daemon/specs/v3/test_v3factory.py

```diff
@@ -372,6 +378,15 @@ def test_name_and_image(self, load_app_config_testdata, factory, filename):
         assert app_spec.name == NAME
         assert app_spec.image == IMAGE
 
+    @pytest.mark.parametrize("filename", (
+            "autoscaling_max_less_than_min",
+    ))
+    def test_autoscaling_max_less_than_min(self, load_app_config_testdata, factory, filename):
+        app_spec = factory(UID, NAME, IMAGE, load_app_config_testdata(filename), ["IO"], ["foo"], "deployment_id", NAMESPACE,
+                           None, None)
+        assert app_spec.autoscaler.min_replicas == TEST_DATA[filename]["autoscaler.min_replicas"]
+        assert app_spec.autoscaler.max_replicas == TEST_DATA[filename]["autoscaler.max_replicas"]
+
```

Comment:
Tests are auto-generated from `TEST_DATA` using `pytest_generate_tests` and this test function https://github.com/fiaas/fiaas-deploy-daemon/blob/0b32bac1158e64a932cccd84dd646924940fa56d/tests/fiaas_deploy_daemon/specs/v3/test_v3factory.py#L444-L452 The generated tests will assert that the resulting app_spec from the file with the name in the `TEST_DATA` key have the defined key+values under that key, so I think this test case is actually already covered by a generated test just by having the entry in `TEST_DATA`. You can try to  remove this test function and remove/re-add the change in factory.py to verify that the generated test breaks